src/Analyzers/CSharp/Tests/RemoveUnusedParametersAndValues/RemoveUnusedParametersTests.cs

```diff
@@ -1153,11 +1153,11 @@ void M3(int {|#4:p5|})
                     """,
                 ExpectedDiagnostics =
                 {
-                    VerifyCS.Diagnostic("IDE0060").WithLocation(0).WithMessage("Remove unused parameter 'p1'"),
-                    VerifyCS.Diagnostic("IDE0060").WithLocation(1).WithMessage("Parameter 'p2' can be removed; its initial value is never used"),
-                    VerifyCS.Diagnostic("IDE0060").WithLocation(2).WithMessage("Remove unused parameter 'p3' if it is not part of a shipped public API"),
-                    VerifyCS.Diagnostic("IDE0060").WithLocation(3).WithMessage("Parameter 'p4' can be removed if it is not part of a shipped public API; its initial value is never used"),
-                    VerifyCS.Diagnostic("IDE0060").WithLocation(4).WithMessage("Parameter 'p5' can be removed; its initial value is never used"),
+                    VerifyCS.Diagnostic("IDE0060").WithSeverity(DiagnosticSeverity.Info).WithLocation(0).WithMessage("Remove unused parameter 'p1'"),
```

Comment:
Unfortunatelly, this is not true for this diagnostic. Here is code that initializes descriptor for it:
https://github.com/dotnet/roslyn/blob/d84d54f2e5d3d66d81453fbd5fafbeeecd0d6cde/src/Analyzers/Core/Analyzers/RemoveUnusedParametersAndValues/AbstractRemoveUnusedParametersAndValuesDiagnosticAnalyzer.cs#L81-L87
If we go-to-def of `CreateDescriptorWithId`, we will see this:
https://github.com/dotnet/roslyn/blob/d84d54f2e5d3d66d81453fbd5fafbeeecd0d6cde/src/Analyzers/Core/Analyzers/AbstractBuiltInCodeStyleDiagnosticAnalyzer_Core.cs#L46-L63
Here severity is initialized as hidden, but it is somewhere overriden to info. So using descriptor won't help much, but will force to make it internal in the analyzer

Reply:
It's being overridden by this:

```
{ CodeStyleOptions2.UnusedParameters, UnusedParametersPreference.NonPublicMethods, NotificationOption2.Suggestion }
```

When you switch to the simpler form, it will go back to using the default severity.